packages/webapp/src/routes/land/any-other-landowners.js

```diff
@@ -13,6 +13,7 @@ const handlers = {
     if (anyOtherLOValue === 'yes') {
       return h.redirect(constants.routes.LANDOWNER_CONSERVATION_COVENANT_INDIVIDUAL_ORGANISATION)
     } else if (anyOtherLOValue === 'no') {
+      request.yar.set(constants.redisKeys.LEGAL_AGREEMENT_LANDOWNER_CONSERVATION_CONVENANTS, null)
```

Comment:
request.yar.clear(constants.redisKeys.LEGAL_AGREEMENT_LANDOWNER_CONSERVATION_CONVENANTS) seems clearer but this works